src/components/Repository.js

```diff
@@ -96,7 +96,7 @@ function Repository({user, match}) {
       .finally(() => setIsForkLoading(false));
   };
 
-  const showFork = repoOwner !== user && user.login ? true : isForkLoading;
+  const showFork = repository && repoOwner !== user && user.login ? true : isForkLoading;
```

Comment:
Fixed this error https://sentry.io/share/issue/4aac93bccae04f929c6605724051260b/